source/tools/airflow.html.md.erb

```diff
@@ -0,0 +1,6 @@
+---
+title: Airflow
+weight: 3
+---
+
+<%= partial 'documentation/tools/airflow' %>
```

Comment:
Why all the `partial` stuff..? Why not just put the content here anyway..? Just wondering... :+1:

Reply:
Agreed, but that looks to be the convention here